keyboards/thumbsup/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Comment:
VIA cannot be enabled in the `default` keymap. Also, the tri-layer feature should not be enabled at the keyboard level.
```suggestion
TRI_LAYER_ENABLE = yes
```